packages/cozy-client/src/types.js

```diff
@@ -92,12 +92,30 @@ import { QueryDefinition } from './queries/dsl'
  */
 
 /**
- * A reference to a document (special case of a relationship used between photos and albums)
```

Comment:
why this removal?

Reply:
Oops, my bad!
I didn't mean to do that..
Maybe just delete the text in parentheses ? That's not true anymore